app/controllers/application_controller.rb

```diff
@@ -3,8 +3,8 @@ class ApplicationController < ActionController::Base
 
   def set_locale
     locale = params[:locale].to_s.strip.to_sym
-    if I18n.available_locales.include? locale
-      I18n.locale = locale
+    I18n.locale = if I18n.available_locales.include? locale
+      locale
     else
       I18n.default_locale
     end
```

Comment:
dùng toán tử 3 ngôi

Reply:
chỗ này e dùng toán tử 3 ngôi thì rubocop nó bắt đổi thành thế này